.gitignore

```diff
@@ -45,3 +45,4 @@ flow-typed/npm/*
 .idea
 npm-debug.log.*
 __snapshots__
+.vscode/settings.json
```

Comment:
I actually would not add this to `.gitignore` -- it's helpful for other contributors using VSCode.